delphi-frontend/src/test/resources/au/com/integradev/delphi/preprocessor/Expressions.pas

```diff
@@ -0,0 +1,67 @@
+unit Expressions;
+
+interface
+
+implementation
+
+{$if 123 = {$i NestedInclude.inc}}
+{$endif}
+
+{$if (*$if True*) False (*$endif*)}
+{$endif}
+
+{$if 123 = {$ifdef FOO} 123 {$else} 321 {$endif}}
+{$endif}
+
+{$if
+  123 // }
+  = { }
+  321 (* } *)
+}
+{$endif}
+
+{$if
+  {$if
+    123 // }
+    = { }
+    321 (* } *)
+  }
+  {$endif}
+}
+{$endif}
+
+const
+  Bracket = '}';
+
+{$if
+  {$if Bracket = '}'}
+  True
+  {$endif}
+}
+{$endif}
+
+{$if
+  {$if Bracket = '}'}
+  True
+  {$endif}
+}
+{$endif}

```

Comment:
This is a duplicate of the previous example.

Reply:
Oops, I probably meant to do a variant with `(*$if ... *)`

Reply:
Added extra test cases for the `(*` bracket style.